scripts/delete_latest_tickets.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 today=`date -I`
-source_file="data/latest-tickets-to-purge"
+source_file="latest-tickets-to-purge-${today}"
 echo "source: ${source_file}"
 
 
 if [ "${DRY_RUN+1}" ] ; then
```

Comment:
Can we fix indentation issues? There are some lines which have tabs instead of 2 spaces.

Reply:
Resolved as suggested